doc/changes/changes_1.3.1.md

```diff
@@ -0,0 +1,31 @@
+# Parquet for Java 1.3.1, released 2022-06-07
+
+Code name: Dependency updates / CVE fixes
+
+## Summary
+
+Dependency updates
```

Comment:
I remember asked us to use more specific names for the release summary. Maybe better:
```suggestion
Compile dependency updates for guava, hadoop-client and parquet-hadoop, along with updates for test and plugin packages.
```